ruby3.2-jrjackson.yaml

```diff
@@ -80,3 +52,27 @@ update:
     identifier: guyboertje/jrjackson
     strip-prefix: v
     use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - busybox
+        - jruby-9.4
+        - jruby-9.4-default-ruby
```

Comment:
```suggestion
```

Reply:
I don't think you need default ruby here right since you're using jruby for the test